Makefile

```diff
@@ -3,11 +3,14 @@ FONT = AvenirNext-Medium.ttf
 
 
 .PHONY: all
-all: sanctuary-logo.png sanctuary-logo.svg
+all: sanctuary-logo.png sanctuary-favicon.png sanctuary-logo.svg
```

Comment:
Would you mind switching `sanctuary-logo.png` and `sanctuary-favicon.png` above and in the target definitions below, to preserve alphabetical order? :smile: